lib/WP_Auth0_DBManager.php

```diff
@@ -24,41 +24,12 @@ public function check_update() {
 		}
 	}
 
-	public function install_db( $version_to_install = null, $app_token = '' ) {
```

Comment:
Not a breaking change, PHP does not report warnings/errors for extra function parameters